docs/operator/k8s_guide.md

```diff
@@ -0,0 +1,725 @@
+## Rucio Auth
+[Reference Helm chart](https://gitlab.cern.ch/rucio-it/flux-compass/-/blob/master/sync/rucio-servers-auth.yaml?ref_type=heads).
+
+>The authentication component of Rucio is responsible for securely verifying the identities of users and services attempting to access the Rucio system. It ensures that only authorised entities can perform actions within the system.
+
+As for the servers, four steps are needed to deploy the authentication pod:
+1. Produce the Helm chart
+2. Setup the LB
+3. Produce the certificates related to the landb-alias used
+4. Add the certificates as secrets and mount them on the k8s pods.
+
+### The `httpd` configuration block
+This block maps the certificate to an account name.
+Setting up `x509` auth can be very tricky, please make sure that the [`grid_site_enabled`](https://github.com/rucio/containers/blob/b51bbceb5aab0a1e07d48845f295cbbb175bdcb9/server/rucio.conf.j2#L104) parameter is set on `True`. 
+
+This enables the `auth/x509_proxy` endpoint!
+
+## Rucio Daemons
+[Reference Helm chart](https://gitlab.cern.ch/rucio-it/flux-compass/-/blob/master/sync/rucio-daemons.yaml?ref_type=heads).
+
+>Daemons in Rucio are background processes that perform various maintenance and operational tasks. They ensure that the system operates smoothly and that data management policies are enforced continuously.
+
+The description of the various daemons can be found in [here](https://rucio.github.io/documentation/bin/rucio-abacus-account).
+### FTS renewal and delegation
+
+> [FTS](https://cs3mesh4eosc.eu/technologies/file-transfer-service-fts) is an open source software for reliable and large-scale data transfers. It provides easy user interfaces for submitting transfers: Python CLI, Python Client, WebFTS and Web Monitoring. Checksums and retries are provided per transfer and it is a flexible tool due to its multiprotocol support (Webdav/https, GridFTP, xroot, SRM). It also allows parallel transfers optimization to get the most from network without burning the storages. 
+#### How does FTS work: users' x509 certs
+
+1. Users will have to pass their own x509 cert, and this is provided by the rucio config.
+2. Then, the x509 is used to map the cert to an identity that must be created within rucio, which corresponds to a user.
+#### How does FTS work: service x509 cert
+
+It is not the user's account that uploads, downloads, etc, but it's a ***service account*** that is set up normally by the collaboration or the team. 
+
+Together with the service account, we also need the corresponding user <u>grid cert</u> that is split in `cert` and `key` and is passed to the `ftsRenewal` daemon, that renews it periodically through a [specific script](https://github.com/rucio/containers/blob/master/fts-cron/renew_fts_proxy.sh.j2):
+```yaml
+- secretName: fts-cert
+  mountPath: /opt/rucio/certs/
+- secretName: fts-key
+  mountPath: /opt/rucio/keys/
+```
+
+#### How to connect to the proper VO?
```

Comment:
Should link to https://github.com/rucio/documentation/blob/main/docs/operator/multi_vo_rucio.md and vice-versa

Reply:
Done.